### ISSUE TITLE
Support for Lambda Runtime java21

### ISSUE THREAD

**Issue by hawflau** [aws-cloudformation/cfn-lint#2960]
### Is this feature request related to a new rule or cfn-lint capabilities?

rules

### Describe the feature you'd like to request

Hi, could you please add `java21` as a supported Lambda Runtime?

```
E3030 You must specify a valid value for Runtime (java21). Valid values are ["dotnet6", "dotnetcore1.0", "dotnetcore2.0", "dotnetcore2.1", "dotnetcore3.1", "go1.x", "java11", "java17", "java8", "java8.al2", "nodejs", "nodejs10.x", "nodejs12.x", "nodejs14.x", "nodejs16.x", "nodejs18.x", "nodejs20.x", "nodejs4.3", "nodejs4.3-edge", "nodejs6.10", "nodejs8.10", "provided", "provided.al2", "provided.al2023", "python2.7", "python3.10", "python3.11", "python3.12", "python3.6", "python3.7", "python3.8", "python3.9", "ruby2.5", "ruby2.7", "ruby3.2"]
```

https://aws.amazon.com/about-aws/whats-new/2023/11/aws-lambda-support-java-21/

### Describe the solution you'd like

Add `java21`

### Is this something that you'd be interested in working on?

- [ ] 👋 I may be able to implement this feature request

### Would this feature include a breaking change?

- [ ] ⚠️ This feature might incur a breaking change

**Comment by kddejong** [comment]
Automatically included with #2951

**Comment by kddejong** [comment]
In release v0.83.4
